### ISSUE TITLE
Is mapkit not supported by the watch simulator ?? 

### ISSUE THREAD

**Issue by ijl0322** [uchicago-mobi/MPCS51032-2017-Spring-Forum#23]
I was trying to display a map on my watchOS app, the map will only show an empty grid and a pin. 
I downloaded the WatchKit Catalog and also can't get the map to display properly, then I found these posts on stackoverflow about the watch simulator not being able to display the map. Not sure if this is true ? 

http://stackoverflow.com/questions/36910019/map-tiles-dont-load-in-xcode-7-3-watch-simulator
http://stackoverflow.com/questions/36164365/wkinterfacemap-does-not-load-map

**Comment by tabinks** [comment]
@ijl0322 It used to be able to.  I am investigating now...will report back in a bit.

**Comment by tabinks** [comment]
I can verify that the simulator does not properly show the map view on watchOS3.  This is frustrating because the same exact code works on watchOS2.  There is a radar filed with Apple about it.

I have added a new playground project (https://github.com/uchicago-mobi/mcps51032-2017-spring-playground/tree/master/2017-Session5-WatchThisMap) that shows an example of a map view in watchOS.  This code works on a real watch (I tested it).  I also added code to show current users location on  the watch (but it is commented out).

If anyone would like to test on a real watch, we can try it in class or office hours.
